CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 Changelog
 =========
 
+Version 4.4.0-beta1 (Nov 28, 2018)
+==================================
+
+ * [Fix](https://github.com/segmentio/analytics-android/pull/574): Send application build number as a string to match our documentation and other libraries. If you were relying on this field being a number, you may need to update reports that rely on the now deprecated behaviour.
+
+ * [New](https://github.com/segmentio/analytics-android/pull/595): Add `Options#putContext` method. This allows you to send context fields for a single event. This can also override default context fields for a single event.
+
+ ```java
+ analytics.track("My Event", properties, new Options().putContext("custom_context_field", true));
+ ```
+
+ * [New](https://github.com/segmentio/analytics-android/pull/582): Send a custom user agent on HTTP requests originating the library. The new user-agent will be formatted as `analytics-android/{version}`.
+
+ * [Fix](https://github.com/segmentio/analytics-android/pull/578): Retry network requests when server responds with a HTTP 429 response code.
+
+ * [Fix](https://github.com/segmentio/analytics-android/pull/575): Update maximum message limit to be 32kb to match our API limits.
+
+ * [Fix](https://github.com/segmentio/analytics-android/pull/573): Fix race condition that could cause events to pick up user IDs after the event was originally recorded.
+
```

Comment:
Should this https://github.com/segmentio/analytics-android/issues/481 be included?

Reply:
Yep - it's the first one I called out (as it's somewhat of a breaking change).

Reply:
Oh I labelled it incorrectly though, fixing now.

Reply:
Actually sorry, #574 is the right PR, #481 is the original issue but not the PR.

Reply:
LGTM. May be useful to add a comment in the original issues about this version in case anyone is watching them for a fix.

Reply:
Added a note!